recipes/brainglobe/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "brainglobe" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/brainglobe-{{ version }}.tar.gz
+  sha256: 8137cfefd448d9ee140127fc9d170647735cb3006adf8871e5348719dfff2aa4
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python
+    - setuptools >=45
+    - wheel
+    - setuptools-scm >=6.2
+    - pip
+  run:
+    - python
+    - napari
+    - bg-space >=0.5.0
+    - cellfinder-core >=0.4.0  # [unix]
+    - cellfinder-core >=0.3.0, <0.4.0  # [win]
```

Comment:
Is this only because of the version reporting issue? A package being noarch would drastically reduce maintenance.

Reply:
It's a deeper issue to do with tensorflow versions that are available on conda channels: https://github.com/conda-forge/cellfinder-core-feedstock/issues/13.

A noarch build with a `cellfinder-core >=0.4.0` [fails on Windows](https://github.com/conda-forge/staged-recipes/runs/14497878183) because of the issue above. [We're looking into how best to handle this going forward]https://github.com/brainglobe/BrainGlobe/issues/27).